src/libtriton/CMakeLists.txt

```diff
@@ -21,9 +21,6 @@ endif()
 if(CMAKE_COMPILER_IS_GNUCXX OR ${CMAKE_CXX_COMPILER_ID} MATCHES "Clang")
     # Flags
     set(CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} -Wall -Wextra -Werror -Wno-unused-parameter -Wno-unknown-pragmas -Wno-mismatched-tags")
-    if(NOT GCOV)
-        set(CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} -O3")
-    endif()
```

Comment:
You don't want to optimize code? I guess you are right for debug build but you should set this flag for release build.

I guess it could be done with CMAKE_CXX_FLAGS_RELEASE

Reply:
CMake by default sets them by default to O3 in Release mode (at least on my system).

You can see that by typing `make VERBOSE=1`. On my system I get 

`[  6%] Building CXX object src/libtriton/CMakeFiles/triton.dir/api/api.cpp.o
cd /home/tarracht/source/Triton-release/src/libtriton && /usr/bin/c++    -I/home/tarracht/source/Triton/src/libtriton/includes -I/home/tarracht/source/Triton-release/src/libtriton/includes  -Wall -Wextra -Werror -Wno-unused-parameter -Wno-unknown-pragmas -Wno-mismatched-tags -fno-stack-protector -fomit-frame-pointer -fno-strict-aliasing -fPIC -std=c++11 -O3 -DNDEBUG   -o CMakeFiles/triton.dir/api/api.cpp.o -c /home/tarracht/source/Triton/src/libtriton/api/api.cpp`